examples/afl_frida/Makefile

```diff
@@ -0,0 +1,23 @@
+ifdef DEBUG
```

Comment:
it is GMake specific so,
mv Makefile GNUmakefile
please.

Reply:
done